mono/metadata/mono-security-windows-uwp.c

```diff
@@ -62,17 +62,20 @@ ves_icall_System_Security_Principal_WindowsImpersonationContext_SetCurrentToken
 }
 
 MonoBoolean
-ves_icall_System_Security_Principal_WindowsImpersonationContext_RevertToSelf (MonoError *error))
+ves_icall_System_Security_Principal_WindowsImpersonationContext_RevertToSelf (MonoError *error)
 {
 	mono_security_win_not_supported ("RevertToSelf", error);
 	return FALSE;
 }
 
 gint32
-mono_security_win_get_token_name (gpointer token, gunichar2 ** uniname, MonoError *error)
+mono_security_win_get_token_name (gpointer token, gunichar2 ** uniname)
 {
+	ERROR_DECL (error); // FIXMEcoop
 	// FIXME This is now supported by UWP.
-	mono_security_win_not_supported ("GetTokenInformation", error);
+	mono_security_win_not_supported ("GetTokenInformation", &error);
```

Comment:
Any example showing how this could adopt alternative pattern? This implementation just followed the implementation of corresponding method in mono-security.c and mono-security-windows.c that also uses the mono_error_set_pending_exception for this method implementation, so in order for this to change I believe we need to adjust all the implementations so they will take a MonoError *error as input parameter, similar to other icalls in those file, correct?

Reply:
These icalls might not be in the right form yet. I’ll look later.